www/source/partials/docs/_glossary-scaffolding.html.md.erb

```diff
@@ -14,6 +14,10 @@ While we are targeting many platforms for automated scaffolding we currently sup
 
 Scaffoldings provide certain overrideable variables for language-specific behavior.  Please see the appropriate scaffolding documentation for details.
 
+## Overriding scaffolding callbacks
```

Comment:
Nit: I'd probably capitalize this heading just for consistency with the other sections.